pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTopicPolicies.java

```diff
@@ -449,7 +449,8 @@ private class GetRetention extends CliCommand {
         private boolean applied = false;
 
         @Parameter(names = { "--global", "-g" }, description = "Whether to get this policy globally. "
-                + "If set to true, broker returned global topic policies")
+                + "If set to true, the broker returns global topic policies"
```

Comment:
Nit: this line is missing a period and a trailing space. When the current annotation is output for the `bin/pulsar-admin topics` help command, it will have `policiesIf` instead of `policies. If`

Reply:
Nice catch!